css/styles.css

```diff
@@ -477,9 +478,10 @@ input[type=checkbox]:checked:after {
 .member {
     display: flex;
     align-items: center;
-    margin: auto;
     flex-direction: column;
-    padding: 5px 70px 0;
+    padding: 2rem 1.5rem;
+    border-radius:17px;
+    box-shadow:5px 5px 20px black;
```

Comment:
If we could math the style from the other 2 cards we have (home page and jokes), I think the site would look more cohesive.

replace these:
```css
border-radius:17px;
box-shadow:5px 5px 20px black;
```

with:
```css   
background: rgba(0, 0, 0, 0.08);
box-shadow: 0 4px 4px rgba(0, 0, 0, 0.5);
border-radius: 20px;
```

Reply:
I bealive it also can be a good idea. Moreover, the cards of members in the team section now have a tiny animation when hover, where it was used transition